### PR TITLE
Release 4.1.0

### PR DESCRIPTION
**Pull request by sujithsojan**

### CODE REVIEW

File: src/app/download-manager/download-manager.page.ts [hunk 1]
```diff
@@ -174,8 +174,6 @@ export class DownloadManagerPage implements DownloadManagerPageInterface, OnInit
           this.downloadedContents = data;
         });
       })
```
Comment (subranil) [inline src/app/download-manager/download-manager.page.ts:176]: Put semicolon at the end

File: src/app/enrolled-course-details-page/enrolled-course-details-page.ts [hunk 2]
```diff
@@ -328,8 +328,6 @@ export class EnrolledCourseDetailsPage implements OnInit, OnDestroy, ConsentPopo
       .then(response => {
         this.baseUrl = response;
       })
-      .catch((error) => {
```
Comment (subranil) [inline src/app/enrolled-course-details-page/enrolled-course-details-page.ts:331]: same as before put semicolon afte then clock closure

File: src/app/enrolled-course-details-page/enrolled-course-details-page.ts [hunk 3]
```diff
@@ -854,8 +851,7 @@ export class EnrolledCourseDetailsPage implements OnInit, OnDestroy, ConsentPopo
                 this.batchExp = false;
               }
             })
-            .catch((error) => {
```
Comment (subranil) [inline src/app/enrolled-course-details-page/enrolled-course-details-page.ts:857]: put semicolon

File: src/app/enrolled-course-details-page/enrolled-course-details-page.ts [hunk 4]
```diff
@@ -903,8 +899,7 @@ export class EnrolledCourseDetailsPage implements OnInit, OnDestroy, ConsentPopo
           this.batchDetails.creatorDetails.firstName = serverProfile.firstName ? serverProfile.firstName : '';
           this.batchDetails.creatorDetails.lastName = serverProfile.lastName ? serverProfile.lastName : '';
         }
-      }).catch(() => {
```
Comment (subranil) [inline src/app/enrolled-course-details-page/enrolled-course-details-page.ts:906]: put semicolon

File: src/app/manage-learn/domain-ecm-listing/domain-ecm-lsiting/domain-ecm-lsiting.component.ts [hunk 5]
```diff
@@ -54,9 +54,7 @@ export class DomainEcmLsitingComponent {
           .then((successData) => {
             this.generalQuestions = successData;
           })
```
Comment (subranil) [inline src/app/manage-learn/domain-ecm-listing/domain-ecm-lsiting/domain-ecm-lsiting.component.ts:56]: put semicolon at the end of then block

File: src/app/manage-learn/ecm-listing/ecm-listing.page.ts [hunk 6]
```diff
@@ -59,10 +56,7 @@ export class EcmListingPage implements OnInit {
           .then((successData) => {
             this.generalQuestions = successData;
           })
```
Comment (subranil) [inline src/app/manage-learn/ecm-listing/ecm-listing.page.ts:58]: put semicolon at the end of then block

File: src/app/manage-learn/image-listing/image-listing/image-listing.component.ts [hunk 7]
```diff
@@ -58,7 +58,6 @@ export class ImageListingComponent implements OnInit {
         this.selectedEvidenceName = this.currentEvidence['name'];
         this.getAllImages();
       })
```
Comment (subranil) [inline src/app/manage-learn/image-listing/image-listing/image-listing.component.ts:60]: put semicolon at the end of then block

File: src/app/manage-learn/image-listing/image-listing/image-listing.component.ts [hunk 8]
```diff
@@ -212,15 +211,13 @@ export class ImageListingComponent implements OnInit {
           this.fileTransfer.create();
           this.cloudImageUpload();
         })
```
Comment (subranil) [inline src/app/manage-learn/image-listing/image-listing/image-listing.component.ts:213]: put semicolon at the end of then block

File: src/app/manage-learn/image-listing/image-listing/image-listing.component.ts [hunk 9]
```diff
@@ -212,15 +211,13 @@ export class ImageListingComponent implements OnInit {
...
-        .catch((err) => {});
     } else {
       this.file
         .checkDir(this.file.externalDataDirectory, 'images')
         .then((success) => {
           this.cloudImageUpload();
           this.fileTransfer.create();
         })
```
Comment (subranil) [inline src/app/manage-learn/image-listing/image-listing/image-listing.component.ts:220]: put semicolon at the end of then block

File: src/app/manage-learn/observation/observation-detail/observation-detail.component.ts [hunk 10]
```diff
@@ -275,9 +274,6 @@ export class ObservationDetailComponent implements OnInit {
             this.goToEcm(sId, e.name);
           }
         })
```
Comment (subranil) [inline src/app/manage-learn/observation/observation-detail/observation-detail.component.ts:276]: put semicolon at the end of then block

File: src/app/manage-learn/observation/observation-detail/observation-detail.component.ts [hunk 11]
```diff
@@ -275,9 +274,6 @@ export class ObservationDetailComponent implements OnInit {
...
-        .catch((error) => {
-          console.log(error);
```
Comment (subranil) [inline src/app/manage-learn/observation/observation-detail/observation-detail.component.ts:279]: put semicolon at the end of then block

**Review by subranil — CHANGES_REQUESTED** [review]

**Review by subranil — APPROVED** [review]

**Review by subranil — APPROVED** [review]

**Review by swayangjit — APPROVED** [review]

**Pull request merged**